.changelog/10091.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+cli: snapshot inspect command would panic on invalid input. [[GH-10091](https://github.com/hashicorp/consul/pull/10091)]
```

Comment:
Doesn't the changelog-build tool auto-add the `[GH-xxxx]` part already?

Reply:
Oh, I didn't know that - some examples I saw still have that explicitly but good to know. I'll remove them.